docs/faq.md

```diff
@@ -5,65 +5,67 @@ description: Get to know more about Next.js with the frequently asked questions.
-
-  <p>We’re ecstatic about both the developer experience and end-user performance, so we decided to share it with the community.</p>
+  <summary>Is Next.js production ready?</summary>
+  <p>Yes! Next.js is used by many of the top websites in the world. See the
+  <a href="/showcase">Showcase</a> for more info.</p>
 </details>
 
 <details>
-  <summary>How big is it?</summary>
-  <p>The client side bundle size should be measured in a per-app basis. A small Next main bundle is around 65kb gzipped.</p>
-</details>
-
-<details>
-  <summary>How can I change the internal webpack configs?</summary>
-  <p>Next.js tries its best to remove the overhead of webpack configurations, but for advanced cases where more control is needed, refer to the <a href="/docs/api-reference/next.config.js/custom-webpack-config.md">custom webpack config documentation</a>.</p>
-</details>
-
-<details>
-  <summary>What syntactic features are compiled? How do I change them?</summary>
-  <p>We track V8. Since V8 has wide support for ES6 and async and await, we compile those. Since V8 doesn’t support class decorators, we don’t compile those.</p>
-
-  <p>See the documentation about <a href="/docs/advanced-features/customizing-babel-config.md">customizing babel config</a> for more information.</p>
+  <summary>How do I fetch data in Next.js?</summary>
+  Next.js provides a variety of methods depending on your use case. You can use:
+  <ul>
+    <li> Client-side rendering: Fetch data with <a href="/docs/basic-features/data-fetching/client-side.md#client-side-data-fetching-with-useeffect">useEffect</a> or <a href="/docs/basic-features/data-fetching/client-side.md#client-side-data-fetching-with-swr">SWR</a> inside your React components</li>
+    <li> Server-side rendering with <a href="/docs/basic-features/data-fetching/get-server-side-props.md">getServerSideProps</a></li>
+    <li> Static-site generation with <a href="/docs/basic-features/data-fetching/get-static-props.md">getStaticProps</a></li>
+    <li> Incremental Static Regeneration by <a href="/docs/basic-features/data-fetching/incremental-static-regeneration.md">adding the `revalidate` prop to getStaticProps</a></li>
+  </ul>
+  To learn more about data fetching, visit our <a href="/docs/basic-features/data-fetching/overview.md">data fetching documentation</a>.
 </details>
 
 <details>
-  <summary>Why a new Router?</summary>
-  Next.js is special in that:
+  <summary>Why does Next.js have its own Router?</summary>
+  Next.js ships with a built-in router for a few reasons:
   <ul>
-    <li>Routes don’t need to be known ahead of time, We don't ship a route manifest</li>
+    <li>It uses a file-system based router which reduces configuration</li>
+    <li>It supports shallow routing which allows you to change the URL without running data fetching methods</li>
+    <li>It can avoid client-side routes manifest since routes don’t need to be known ahead of time </li>
```

Comment:
```suggestion
```
We do have client-side routes manifests now although they do not block navigation. Might be good to omit this item.